src/components/search-panel/SearchPanel.tsx

```diff
@@ -32,6 +32,12 @@ const SearchPanel = () => {
 	const activeWorkspace = useActiveWorkspace();
 	const { ref: searchPanelRef } = useActivePanel(null);
 	const searchStore = useSearchStore();
+	const messagesStore = useMessagesWorkspaceStore();
+	const {
+		filterStore: {
+			messagesFilter: { streams },
```

Comment:
i think you should have taken messageSessions from messages store for autocomplete list